src/codeImprovements/calculateFactorial.js

```diff
@@ -0,0 +1,22 @@
+//Improve this function
+/**function calculateFactorial(n) {
+  if (n === 0 || n === 1) {
+    return 1;
+  } else {
+    var factorial = 1;
+    for (var i = 2; i <= n; i++) {
+      factorial *= i;
+    }
+    return factorial;
+  }
+} */
+
+function calculateFactorial(number){
+    let result = 1;
```

Comment:
its better if you put this result under that if statement that is checking if a number is less 2 so that if the number is less than 2 you dont create an extra variable you are not going to use